src/tests/framework/Framework_Monitor_002.test.js

```diff
@@ -10,7 +10,16 @@ export default {
     description:
       'Tests if the Framework Monitor module return measurement values for WebKitBrowser',
     setup() {
-      pluginActivate.call(this, constants.webKitBrowserPlugin)
+      return this.$sequence([
+        () => pluginDeactivate.call(this, 'WebKitBrowser'), //make sure the browser is turned off
+        () => pluginDeactivate.call(this, 'UX'), //make sure UX is turned off
+        () => pluginDeactivate.call(this, 'Netflix'), //make sure Netflix is turned off
+        () => pluginDeactivate.call(this, 'Cobalt'), //make sure Cobalt is turned off
+        () => pluginActivate.call(this, 'WebKitBrowser'),
```

Comment:
I dont think this is needed for a monitor test. We dont need to do the webkit/ux/netflix/webkit cycle here.